rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/logsdb/10_settings.yml

```diff
@@ -0,0 +1,394 @@
+---
+setup:
+  - skip:
+      known_issues:
+        - cluster_feature: "gte_v8.15.0"
```

Comment:
This stuff does not work for anything above 8.12 i think.

Reply:
I will double check but I see it used in other places too. What I know is that we are still not able to get rid of this feature until the full support for testing skipping is complete. Talking to people in core-infra it looks like we will keep this way of filtering for a while before we completely drop it.

Reply:
I didn't notice that this is inside `known_issues`. I wonder if can use new `capabilities` thing and add a capability `logs_index_mode` to create index API. That seems in line with latest changes in skipping.

Reply:
Using `known_issues` feels a bit like a workaround here :)